packages/radix-ui-themes/src/components/tabs.css

```
@@ -134,7 +139,7 @@
     bottom: 0;
     left: 0;
     right: 0;
-    background-color: var(--accent-10);
+    background-color: var(--tabs-list-active-accent);
```

Nit:

We usually opt to use CSS variables almost as the last resort to handle multiple style dependencies related to the same value across different parts of the component.

Otherwise, we’d rather trade off larger selector complexity for being able to read the values right where they are applied, like this:

```suggestion
    background-color: var(--accent-10);
    
    :where(.rt-TabsTrigger.rt-high-contrast) & {
      background-color: var(--accent-12);
    }
  }
```

---

Good to know, thank you!